### PR TITLE
Create A Schema For The Plant & Insert data For Plants Into DB

### PR DESCRIPTION
**Pull request by ameirahassoun**
@JackMiszencin @abdulazeeznaji

### CODE REVIEW

File: Greenify-BackEnd/insertAllPlants.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+var db = require('./db/index.js')
+var data = require('./data/plants.json')
+// we should run insertAllPlants.js as server with node ...
+var insertAllPlants = function() {
+	console.log(data);
```
Comment (abdulazeeznaji) [inline Greenify-BackEnd/insertAllPlants.js:5]: Console messages shouldn't be committed

File: Greenify-FrontEnd/App.test.js [hunk 2]
```diff
@@ -1,9 +1,13 @@
+import 'react-native'
 import React from 'react';
 import App from './App';
```
Comment (abdulazeeznaji) [inline Greenify-FrontEnd/App.test.js:4]: Unnecessary space

File: Greenify-BackEnd/insertAllPlants.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+	 db.Plant.create(data, function(err, data) {
+    db.disconnect();
```
Comment (abdulazeeznaji) [inline Greenify-BackEnd/insertAllPlants.js:7]: Console messages shouldn't be committed

**Review by abdulazeeznaji — COMMENTED** [review]
Just review the comments --> Comments

File: .gitignore [hunk 4]
```diff
@@ -57,3 +57,6 @@ typings/
 # dotenv environment variables file
 .env
 
+# package-lock.json
```
Comment (JackMiszencin) [inline .gitignore:60]: Why the extra, commented-out line?

**Pull request merged**